rotary/rules/rotary.smk

```diff
@@ -350,12 +350,12 @@ rule assembly_flye:
 # TODO - the math for memory per thread should really be done somewhere other than 'resources' - what is best practice?
 rule assembly_end_repair:
     input:
-        qc_long_reads="{sample}/qc_long/nanopore_qc.fastq.gz",
-        assembly="{sample}/assembly/flye/assembly.fasta",
-        info="{sample}/assembly/flye/assembly_info.txt"
+        qc_long_reads="{sample}/qc_long/{sample}_nanopore_qc.fastq.gz",
+        assembly="{sample}/assembly/flye/{sample}_assembly.fasta",
+        info="{sample}/assembly/flye/{sample}_assembly_info.txt"
     output:
-        assembly="{sample}/assembly/end_repair/repaired.fasta",
-        info="{sample}/assembly/end_repair/repaired_info.tsv",
+        assembly="{sample}/assembly/end_repair/{sample}_repaired.fasta",
```

Comment:
Like Flye, my code also doesn't allow you to specify the files names in the output folder. So maybe best to keep these as `repaired.fasta` and `repaired_info.tsv`? They will be symlinked below anyway, so the name could be adjusted then.

Reply:
@jmtsuji Added mv commands to address this issue during my debugging.